crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -75,6 +75,52 @@ where
     pub wr: W,
 }
 
+fn replace_close_inline_script(raw: &str) -> String {
```

Comment:
This is still unconditional. Can we check for the existence of the pattern and return `Cow::Borrowed(raw)` if it's not found?